src/utils/ExtractTestResults.ts

```diff
@@ -22,7 +25,8 @@ import { ValidationUtil } from './ValidationUtil';
  * @param record
  */
 export const extractMCTestResults = (record: DynamoDBRecord): MCRequest[] => {
-  const testResultUnmarshall = unmarshall(record.dynamodb.NewImage as any);
+  // @ts-ignore
```

Comment:
Why has a ts ignore been added?

Reply:
type error on 29. It was a working implementation from elsewhere however here it was not having it

Reply:
The @tes-ignore could be masking a potential issue. 

Can you look a bit more into why your implementation is not working in the same way as others please? (pull-test-results for example)